libs/extensions/std/exec/src/http-extractor-executor.ts

```diff
@@ -126,6 +126,17 @@ export class HttpExtractorExecutor extends AbstractBlockExecutor<
           );
         }
 
+        if (responseCode === 302) {
```

Comment:
301 is also a redirect code I assume.

Reply:
Changed to >= 301, as 300 is used for 'multiple choices'. The remaining `300 < response codes < 400` point to redirect location or are not used in get context.

Reply:
ok, done